imager/ImagerProfile/models.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+class ActiveImagerManager(models.Manager):
+    def get_queryset(self):
+        return super(ActiveImagerManager,
+                     self).get_queryset().filter(user__is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    """Imager profile that is attached to user on creation"""
+    user = models.OneToOneField(User, related_name='profile')
+
+    profile_picture = models.ImageField(null=True, blank=True,
+                                        upload_to='images')
+
```

Comment:
in general it's better, if you are going to break *some* params onto a second line, to break *each* onto an individual line.  In this case that's probably not required, as you could do this:

```python
profile_picture = models.ImageField(
    null=True, blank=True, upload_to='images'
)
```